src/main/java/dao/Edit.java

```diff
@@ -0,0 +1,57 @@
+package dao;
```

Comment:
Please follow the right convention for the package name. For example, you are creating the application for HotWax Systems so package name will be "com.hotwaxsystems.dao" or any other better name.

https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html